extensions/amp-story/validator-amp-story.protoascii

```diff
@@ -151,6 +183,14 @@ tags: {  # <amp-story-grid-layer>
     value: "landscape-half-left"
     value: "landscape-half-right"
   }
+  attrs: {
```

Comment:
please alphabetize based on their names (and yeah, template above should be below position)